dnn/variational_inference.ipynb

```diff
@@ -7,7 +7,7 @@
         "colab_type": "text"
```

Comment:
<p>combine the next 4 code blocks into a single code block "data preparation" -- unless there are major reasons to execute them independently.</p><p>I also bet there is code here that could be put into ddfcommon, but that is a bigger change to do later.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf-isoscapes/pull/176/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>